src/util.js

```diff
@@ -24,6 +24,20 @@ var Util = {
         total: 0,
         message: null
     },
+    
+    countOccurences: function(contentString, searchString) {
+        var count = 0;
+
+        var searchRegex = new RegExp(searchString, "g");
+        var widgetRegex = /^\[\[\u2603 (([a-z-]+) ([0-9]+))\]\]/g;
```

Comment:
This is really similar to Util.rWidgetRule above; could we declare it up there?

Also, what does the combination of `^` and `/g` do? Testing in node, this looks to me like it ignores the `/g`, which I think is probably not what we want here.

Weird things start happening if you search for widget names right now--highlighting doesn't seem to work, while replacement does. I'm guessing that might be related to this.